diesel/src/macros/insertable.rs

```diff
@@ -231,6 +231,7 @@ macro_rules! Insertable_column_expr {
 
 #[cfg(test)]
 #[allow(missing_debug_implementations, missing_copy_implementations)]
+#[cfg_attr(feature = "clippy", allow(result_unwrap_used))]
```

Comment:
Isn't this redundant with https://github.com/diesel-rs/diesel/pull/305/files#diff-7ab3c70a207718b2fce878a8f925a415R22?